### PR TITLE
Car docs: show more information about cars

### LINKED ISSUE #25333: Car docs: show more information about cars
- [x] Show actual steer and enable speeds
- [x] Remove (useless) stars
- [ ] Show extra information (hidden by default or all the time), like the detail sentence
- [ ] Clean up diff bot to use the lines instead
- [x] Add auto-resume from stop star - https://github.com/commaai/openpilot/pull/25572
- [ ] Link harness to shop and auto-select

### PR DESCRIPTION
**Pull request by sshane**
Reference, old car docs: https://github.com/commaai/openpilot/blob/62bb43b7e279d81d03d1023ea24dc1f845fc3608/docs/CARS.md

Preview: https://github.com/commaai/openpilot/blob/bb-stars/docs/CARS.md

### CODE REVIEW

File: selfdrive/car/CARS_template.md [hunk 1]
```diff
@@ -1,30 +1,17 @@
 {% set footnote_tag = '[<sup>{}</sup>](#footnotes)' -%}
-{% set star_icon = '[![star](assets/icon-star-{}.svg)](##)' -%}
 
 <!--- AUTOGENERATED FROM selfdrive/car/CARS_template.md, DO NOT EDIT. --->
 
 # Supported Cars
 
 A supported vehicle is one that just works when you install a comma device. Every car performs differently with openpilot, but all supported cars should provide a better experience than any stock system.
 
-## How We Rate The Cars
```
Comment (sshane) [inline selfdrive/car/CARS_template.md:10]: Maybe a short hand-written description for each would be good. For example what steering torque units are, and what it means/is good. What you get with openpilot ACC vs. Stock ACC

**Comment by adeebshihadeh** [comment]
Let's also add a column for the harness name and ideally link to it on the shop.

File: selfdrive/car/CARS_template.md [hunk 2]
```diff
@@ -1,30 +1,17 @@
...
-{% endfor %}
-
-{% endfor %}
-{% endfor %}
-
 # {{all_car_info | length}} Supported Cars
 
 |{{Column | map(attribute='value') | join('|')}}|
-|---|---|---|:---:|:---:|:---:|:---:|
+|---|---|---|:---:|:---:|:---:|:---:|:---:|
```
Comment (adeebshihadeh) [inline selfdrive/car/CARS_template.md:12]: this should be done automatically

**Comment by adeebshihadeh** [comment]
Any blocker on this? Looks pretty good to me.

**Comment by sshane** [comment]
Still want the torque stars? If not I'll clean up

**Comment by adeebshihadeh** [comment]
Let's do a star for now.

**Review by adeebshihadeh — APPROVED** [review]
Looks great. Let's also add resume from stop at some point; then, we should have all the info the website shows.

**Pull request merged**